handlers/handlers.go

```diff
@@ -15,35 +15,36 @@ import (
 	"github.com/Xa4y322/go_final_project/models"
 )
 
-const DATE_FORMAT string = "20060102"
+const DateFormat string = "20060102"
 
 func nextDate(now time.Time, date string, repeat string) (string, error) {
 	if repeat == "" {
 		return "", errors.New("repeate is empty")
 
-	} else if strings.Contains(repeat, "d ") {
-		days_to_add, err := strconv.Atoi(strings.TrimPrefix(repeat, "d "))
+	}
+	if strings.Contains(repeat, "d ") {
+		DaysToAdd, err := strconv.Atoi(strings.TrimPrefix(repeat, "d "))
 		if err != nil {
 			return "", err
 		}
-		if days_to_add > 400 {
+		if DaysToAdd > 400 {
```

Comment:
локальные переменные лучше с маленькой буквы делать
```suggestion
		if daysToAdd > 400 {
```